### PR TITLE
feat(switch): change lane switch syntax for remote lanes

### PR DESCRIPTION
**Pull request by luvkapur**
## Proposed Changes

- Change `bit switch` syntax for remote lanes from `bit switch stage --remote org.scope` to `bit switch org.scope/stage`

### CODE REVIEW

File: src/api/consumer/lib/switch.ts [hunk 1]
```diff
@@ -26,51 +26,55 @@ export default async function switchAction(
...
-    const remoteLaneObjects = await scopeComponentImporter.importFromLanes([
-      RemoteLaneId.from(laneName, remoteScope as string),
-    ]);
+    const remoteLaneObjects = await scopeComponentImporter.importFromLanes([remoteLaneId]);
     const remoteLaneComponents = remoteLaneObjects[0].components;
     const laneExistsLocally = lanes.find((l) => l.name === switchProps.localLaneName);
     if (laneExistsLocally) {
-      throw new GeneralError(`unable to checkout to a remote lane ${remoteScope}/${laneName}.
+      throw new GeneralError(`unable to checkout to a remote lane ${switchProps.remoteScope}/${switchProps.laneName}.
     the local lane ${switchProps.localLaneName} already exists, please switch to the local lane first by omitting --remote flag
```
Comment (davidfirst) [inline src/api/consumer/lib/switch.ts:58]: This error should now be changed. It should suggest removing the scope-name probably.

File: src/api/consumer/lib/switch.ts [hunk 2]
```diff
@@ -26,51 +26,55 @@ export default async function switchAction(
 
 async function populateSwitchProps(consumer: Consumer, switchProps: SwitchProps) {
   const lanes = await consumer.scope.listLanes();
-  const { laneName, remoteScope } = switchProps;
-  if (remoteScope) {
+  const localLane = lanes.find((lane) => lane.name === switchProps.laneName);
+
+  if (!localLane) {
```
Comment (davidfirst) [inline src/api/consumer/lib/switch.ts:31]: I'd suggest switching the if/else. (`if (localLane) {} else {}`).

Comment (luvkapur) [reply src/api/consumer/lib/switch.ts]: fixed

File: src/api/consumer/lib/switch.ts (hunk 1 above)
Comment (luvkapur) [reply src/api/consumer/lib/switch.ts]: fixed

**Review by davidfirst — APPROVED** [review]

**Pull request merged**